_posts/2020-11-08-basic-coding-applications-for-commercial-strategy.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title:  "Basic Coding Applications for Commercial Strategy"
+date:   2020-11-08 12:54:04 -0700
+categories: sales commercial strategy revenue cogs margins applications
+---
+By, Debbie Vo
+
+Every department for every company can benefit from data analysis. Coding allows large data sets to be manipulated and presented in a helpful way. For all code, the goal is that you start with raw information and you end up with useful information derived from the raw form. In genetics, you may have genome sequences as your starting material that are used to determine the similarities and differences between the DNA for different samples. This analysis has many different applications, including the determination of mutations in cancer cells.
+
+Benefits of programming can also be found outside of research and development such as in sales and marketing. Below are a few examples of how R  code can be used to create impactful tools for commercial departments.
+
+
+# Data Cleaning
```

Comment:
I like how you clearly listed the two main things we need to check in the csv files we want to read in. It might be beneficial for the first-time user if you could include an intro on the use of tidyverse tools. Describing the ones that will be utilized.